test/controllers/benefits/CompanyBenefitsSummaryControllerSpec.scala

```diff
@@ -104,7 +104,8 @@ class CompanyBenefitsSummaryControllerSpec extends BaseSpec {
     mcc,
     inject[CompanyBenefitsView],
     mockJourneyCacheRepository,
-    inject[ErrorPagesHandler]
+    inject[ErrorPagesHandler],
+    mockEmpIdCheck
```

Comment:
Should not we have the case for emp id check not found, and the backend services are not called for all the controller specs